src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/QueryClient.java

```diff
@@ -19,6 +24,4 @@ public QueryClient(QueryClientOptions config) {}
    * @param queryKey
    */
   public native void invalidateQueries(List<?> queryKey);
-
-  public native void clear();
```

Comment:
`clear` removes listeners, which leads to all sorts of bugs, so we pretty much never want to use this.